### ISSUE TITLE
tabs light version

### ISSUE THREAD

**Issue by marcoscv-work** [liferay/clay#5445]
Finally (you guys warned about this). We will need a light version of the tabs:

- Without border-top and bg

Example:
https://codepen.io/marcoscv/full/OJoYXOQ

<img width="333" alt="image" src="https://user-images.githubusercontent.com/7963804/228827364-be58fa96-90bd-4cc0-96f4-1e5f02dbc970.png">

**Comment by pat270** [comment]
@marcoscv-work I know we released the pr with this nav-tabs change:

![nav-tabs](https://user-images.githubusercontent.com/788266/229226591-3ca4ab43-5e1a-41f6-af52-76c3cdd188fe.jpg)

What do you think about reverting it and making nav-tabs-light the default and the one with the gray background nav-tabs-light? It fit's better with the way Clay is constructed, where we add features/styles in the variants instead of removing them.

/cc @drakonux @emiliano-cicero

**Comment by marcoscv-work** [comment]
I think the light version is enough without the need to make it the default version.
Most of the uses of the tabs are fine with the new style in the portal, and the ones that are broken are usually because they have been hacked or the styles have been overridden.
The example you provide is really good, we cannot use 10 tabs in a collapsible line, on those cases the design should be made with a vertical nav

Just a detail, we can see a slight border-buttom:

<img width="269" alt="image" src="https://user-images.githubusercontent.com/7963804/229480027-0e8b5087-36a4-4830-b596-33227cb83902.png">

**Comment by drakonux** [comment]
Hey, @pat270 let's make the default version the one with the grey background. We want to encourage product teams to use this version instead of the "light" one for accessibility. So it's an excellent way to promote the usage of the grey one. 

cc\ @ethib137

**Comment by marcoscv-work** [comment]
Updated codepen:
https://codepen.io/marcoscv/full/OJoYXOQ
<img width="554" alt="image" src="https://user-images.githubusercontent.com/7963804/230049482-45934518-91c3-42b2-ae5b-73e702c3fd44.png">

@pat270 sorry for asking for so many changes, as you already said most cases in DXP are better suited to the light version.
Since we are removing the top underline, I think we don't need a variant, we just optionally add a bg, in the codepen I have created a `bg-light-l1` class but it's up to you if you want to create a new bg or a css variant like `nav-tabs-bg-gray-or-something-like-that`

**Comment by marcoscv-work** [comment]
We can go ahead with the light version as default `nav-tabs`, everything about the gray BG can be done later, no worries about that part
